apps/cli/package.json

```diff
@@ -57,6 +57,7 @@
     "@google-cloud/storage": "^5.18.2",
     "@jupyterlab/nbformat": "^3.3.2",
     "@reduxjs/toolkit": "^1.7.2",
+    "adm-zip": "^0.5.9",
```

Comment:
Don't love adding this dependency, but what we have in jtex is `unzipper` - not great for zipping, hah. Could potentially replace that with this...?